src/actions/tests/__snapshots__/ast.spec.js.snap

```diff
@@ -4,8 +4,8 @@ exports[`ast getOutOfScopeLocations with selected line 1`] = `
 Array [
   Object {
     "end": Object {
-      "column": 16,
-      "line": 1,
+      "column": 63,
```

Comment:
hmm interesting that this changed...

Reply:
this was auto-updated by $npm test -- -u